launch-playground.sh

```diff
@@ -15,6 +15,18 @@ else
   exit
 fi
 
+echo "Checking if MySQL is runing"
+if lsof -Pi :3306 -sTCP:LISTEN -t >/dev/null ; then
+    echo "MySQL may allready be running on port 3306. Please shut down MySQL before launching the playground."
+    exit
```

Comment:
Do we need to exit with code -1, too?

Reply:
I don't think so other existing exit in the script doesn't exit with -1